docs/pages/experiments/index.js

```diff
@@ -32,6 +33,9 @@ export default function Experiments({ experiments }) {
 
   return (
     <React.Fragment>
+      <Head>
+        <meta name="robots" content="noindex,nofollow" />
```

Comment:
Add a no follow to void Google to spend further SEO crawl budget on the dead end links we have on this index page.